modules/ROOT/nav.adoc

```diff
@@ -60,7 +60,7 @@
 * xref:troubleshooting.adoc[Troubleshooting]
  ** xref:insight.adoc[Insight]
  ** xref:viewing-log-data.adoc[View Log Data]
-  *** xref:custom-log-appender.adoc[Custom Log Appender]
+ ** xref:custom-log-appender.adoc[Integrating With Your Logging System Using Log4j]
```

Comment:
Words to Capitalize
When title capitalization is required, capitalize these words:
• Nouns
(Switcher, Migration), including nouns used to modify other nouns (Validation Rules, Verification Methods)
• Adjectives
(Encrypted, Multiple)
• Pronouns
(It, That, You, Your, Who, Which)
• Verbs
(Is, Are, Deliver, Encrypt)
• All words in a verb phrase
(Turn On, Sign In, Log Out)
• Adverbs
(Easily, Quickly)
• Prepositions five or more letters long
(Without, Between)
• The first and last words of a title
No matter what the words are
• The first word after the colon in a title
(Flexible Pages: The Big Picture)
• Words in hyphenated compounds
(Real-Time Quotes, Third-Party Software, Two-Factor Authentication; but Just-in-Time Manufacturing)
• Key names
(Ctrl, Alt, Del; don't use all caps)
• Subordinating conjunctions
(Although, Because, When)
Words Not to Capitalize
When title capitalization is required, don't capitalize these words:
• Articles (a, an, the) and coordinating conjunctions
(and, but, for, no, or, so, yet)
• The "to" in an infinitive
(to Be)
• Prepositions that have fewer than five letters
(to, with, from) and are not part of a verb like "Sign In"